### PR TITLE
chore(web): show installed widgets & fix computed feature property

### PR DESCRIPTION
**Pull request by airslice**
# Overview

This PR:
- Allows show installed widgets.
- Fixs the computed feature property.
- Comment the setting id for legacy layer to avoid crash.

**THIS IS REQUIRED FOR SHOW CASE.**

### CODE REVIEW

File: web/src/services/api/widgetsApi/utils.ts [hunk 1]
```diff
@@ -40,8 +40,9 @@ export const getInstallableWidgets = (rawScene?: GetSceneQuery) => {
       return plugin?.extensions
         .filter(
           e =>
-            e.type === PluginExtensionType.Widget &&
-            AVAILABLE_WIDGET_IDS.includes(`reearth/${e.extensionId}`),
+            (e.type === PluginExtensionType.Widget &&
+              AVAILABLE_WIDGET_IDS.includes(`reearth/${e.extensionId}`)) ||
+            plugin.id !== "reearth",
```
Comment (KaWaite) [inline web/src/services/api/widgetsApi/utils.ts:45]:
```
(e.type === PluginExtensionType.Widget &&
              (AVAILABLE_WIDGET_IDS.includes(`reearth/${e.extensionId}`)) ||
            plugin.id !== "reearth")
```

Comment (KaWaite) [reply web/src/services/api/widgetsApi/utils.ts:45]: This is the widgetsApi, so should be widget extension 👍🏻

Comment (airslice) [reply web/src/services/api/widgetsApi/utils.ts]: Updated!

File: web/src/beta/lib/core/Map/Layers/hooks.ts [hunk 2]
```diff
@@ -365,7 +365,8 @@ export default function useHooks({
         rawLayer.type === "simple" &&
         rawLayer.data?.value &&
         // If data isn't cachable, reuse layer id for performance.
-        DATA_CACHE_KEYS.some(k => !rawLayer.data?.[k])
+        DATA_CACHE_KEYS.some(k => !rawLayer.data?.[k]) &&
+        Object.isExtensible(rawLayer.data.value)
```
Comment (keiya01) [inline web/src/beta/lib/core/Map/Layers/hooks.ts:369]: User need to take care not to pass inextensible object if some problem occurs later.

**Review by KaWaite — APPROVED** [review]

**Pull request merged**